dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -238,13 +248,55 @@ private constructor(
             return this
         }
 
+        /**
+         * Enable the gestures auto tracker. By enabling this feature the SDK will intercept
+         * tap events and automatically send those as RUM UserActions for you.
+         */
+        fun trackGestures(): Builder {
+            rumConfig = rumConfig.copy(trackGestures = true)
+            return this
+        }
+
+        /**
+         * Sets the automatic view tracking strategy used by the SDK.
+         * By default this is NONE.
+         * @param strategy as the strategy to be used by the automatic view tracker.
+         * @see ViewTrackerStrategy.TRACK_ACTIVITIES_AS_VIEWS
+         * @see ViewTrackerStrategy.TRACK_FRAGMENTS_AS_VIEWS
+         * @see ViewTrackerStrategy.NONE
+         */
+        fun trackViews(strategy: ViewTrackerStrategy): Builder {
+            rumConfig = rumConfig.copy(viewTrackerStrategy = strategy)
+            return this
+        }
+
         private fun checkCustomEndpoint(endpoint: String) {
             if (endpoint.startsWith("http://")) {
                 needsClearTextHttp = true
             }
         }
     }
 
+    /**
+     * The available strategies for the automatic View tracker.
+     */
+    enum class ViewTrackerStrategy {
```

Comment:
This is total nit-picking, and you can change that in a future PR if you prefer, but I'd name that `ViewTrackingStrategy`. It is a strategy on how to track views, so it should come with a verb.

Reply:
Will change these names in my next PR